test/parallel/test-fs-promises-file-handle-stream.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+const common = require('../common');
+
+// The following tests validate base functionality for the fs.promises
+// FileHandle.write method.
+
+const fs = require('fs');
+const { open } = fs.promises;
```

Comment:
const {
  readFileSync,
  writeFileSync,
  promise: { open },
} = require('fs');

Reply:
It's a pattern used a lot in `FileHandle` test, unless we change the other occurrences I think we should keep it as it.

https://github.com/nodejs/node/blob/8ef68e66d0465441a79a5dae22e480bf0d83ed25/test/parallel/test-fs-promises-file-handle-append-file.js#L8-L9
https://github.com/nodejs/node/blob/8ef68e66d0465441a79a5dae22e480bf0d83ed25/test/parallel/test-fs-promises-file-handle-chmod.js#L8-L9
https://github.com/nodejs/node/blob/5370220ab2e8978611a827ac11aba1318bb93969/test/parallel/test-fs-promises-file-handle-read.js#L8-L9
https://github.com/nodejs/node/blob/f9447b71a6b458adbb265f8a5fd38ebf41bc97a4/test/parallel/test-fs-promises-file-handle-readFile.js#L8-L14
https://github.com/nodejs/node/blob/330f25ef822dac6ed8a0afb3422bd920726eb43d/test/parallel/test-fs-promises-file-handle-write.js#L8-L9
https://github.com/nodejs/node/blob/52abf271c563ddffdc93b444ea05e5347a7f2784/test/parallel/test-fs-promises-file-handle-writeFile.js#L8-L9